eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -121,10 +122,88 @@ function Confirm-ChangeLogEntry {
   return $true
 }
 
-function Set-TestChangeLog($TestVersion, $changeLogFile, $ReleaseEntry) {
-  Set-Content -Path $changeLogFile -Value @"
-# Release History
-## $TestVersion ($(Get-Date -f "yyyy-MM-dd"))
-- $ReleaseEntry
-"@
+function New-ChangeLogEntry {
+  param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [String]$Version,
+    [String]$Status=$UNRELEASED_TAG,
+    [String]$Title,
+    [String[]]$Content,
+    [String]$IgnoreInvalids=$True
+  )
+
+  # Validate RelaseStatus
+  $Status = $Status.Trim().Trim("()")
+  if ($Status -ne "Unreleased") {
```

Comment:
This is another place we could share the constant if we didn't assume there were parenthesis around the status.